docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -988,31 +1007,40 @@ thor: []
       </sect3>
 
       <sect3 id="FileAccessCosts">
-        <title>File Access Cost</title>
+        <title>File Access Costs</title>
+<para> For logical files, the costs calculated are based on two types of file access.  </para>
+<itemizedlist>
+<listitem>File Access Cost
+</listitem>     
+<listitem>File Cost at Rest
+</listitem>
+</itemizedlist>
+
 
         <para>File access costs are the costs of reading and writing to the
         files. Many storage planes do have a separate charge for data
-        operations. The cost of reading and writing to the file will be
+        operations. The cost of reading and writing to the file is
         included in the file access cost value. Any other cost associated with
         file operations (such as delete or copy) will not be tracked or
         included as part of file access cost at this time.</para>
-
+        
         <para>The file access cost displays as part of the cost field on the
```

Comment:
There are now two separate fields for the file costs now.  The file access costs isn't displayed _as part of_ the cost field.  File access cost is shown as a field. The At rest is shown as separate field.